cluster/charts/rook-ceph/templates/role.yaml

```diff
@@ -32,4 +32,12 @@ rules:
   - create
   - update
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
```

Comment:
@david972 The `endpoints` is declared in `rook-ceph-global` clusterRole in this other ClusterRole:
https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/templates/clusterrole.yaml#L69

This is a new privilege that was just added in master. Did you intend to run a master build? It's recommended to run a release build, such as the latest v0.9.2. I wouldn't expect this change to be needed.

Reply:
You right. 
I deployed the stable charts but I had set master as tag by mistake.